docs/pages/release_notes.rst

```diff
@@ -5,6 +5,21 @@ Release notes
 
 What has been changed, and how to update between versions.
 
+*****************
+Chestnut update (v4.3.0)
+*****************
+
+Release date: 2022-10-27
+
+This update adds forward compatibility with the upcoming v5 release.
+It removes validation of ``legalBase`` from ``NutsAuthorizationCredential``, which was never properly defined in the JSON-LD contexts.
+The upcoming v5 release will refuse to issue credentials with fields that were not defined in the credential's context.
+But, since ``legalBase`` is currently required, this means ``NutsAuthorizationCredentials`` can't be issued any more.
```

Comment:
This leaves me confused, it can't? Oh noes! Or do you mean, without this update?